docs/modules/ROOT/pages/reference/extensions/aws2-ddb.adoc

```diff
@@ -45,3 +45,28 @@ Check the xref:user-guide/index.adoc[User guide] for more information about writ
 This extension auto-enables SSL support in native mode. Hence you do not need to add
 `quarkus.ssl.native=true` to your `application.properties` yourself. See also
 https://quarkus.io/guides/native-and-ssl[Quarkus SSL guide].
+
+== Additional Camel Quarkus configuration
+
+=== Optional integration with Quarkus Amazon DynamoDB
+
+If desired, it is possible to use the Quarkus Amazon DynamoDB Client extension in conjunction with Camel Quarkus AWA2 DDB.
+Note that this optional and not at all mandatory.
```

Comment:
Godo to explicit that. Maybe the formulation: Note that this is totally optional.

Reply:
To my mind `"totally optional" == "optional and not at all mandatory"`.

Happy to change it if necessary though.

Reply:
I felt it like a repetition, but there is no big requirement to change.